OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -283,10 +284,11 @@ void LoadAndDisplayNews(Widget newsBG)
 					if (!fetchedNews)
 					{
 						// Send the mod and engine version to support version-filtered news (update prompts)
-						var newsURL = Game.Settings.Game.NewsUrl + "?version={0}&mod={1}&modversion={2}".F(
+						var newsURL = Game.Settings.Game.NewsUrl + "?version={0}&mod={1}&modversion={2}&sysinfoversion={3}".F(
 							Uri.EscapeUriString(Game.Mods["modchooser"].Metadata.Version),
 							Uri.EscapeUriString(Game.ModData.Manifest.Id),
-							Uri.EscapeUriString(Game.ModData.Manifest.Metadata.Version));
+							Uri.EscapeUriString(Game.ModData.Manifest.Metadata.Version),
+							Uri.EscapeUriString(SystemInformationVersion.ToString()));
```

Comment:
This might be better in the `if` case below.  Not much use knowing this if there isn't any data to go with it.